firstvoices/backend/tasks/import_job_tasks.py

```diff
@@ -211,6 +211,17 @@ def import_job_dry_run(data, import_job_instance, logger):
         import_job=import_job_instance.id,
     )
 
+    # Clearing out old report if revalidate
+    old_report = import_job_instance.validation_report
```

Comment:
What if there is an old report but they don't choose revalidate? Are we intentionally leaving the old report?